sidebars.js

```diff
@@ -25,12 +25,18 @@ const sidebars = {
     {
       type: 'category',
       label: 'Deployment Guides',
-      items: ['cbdb-op-software-hardware',
+      items: [
+      {
+        type: 'category',
+        label: 'Quick Start',
```

Comment:
For the quickstart, it refers to how to install and start some actions, like creating one table, etc. We can modify the label to "Build and Install" or something like this to make it more clear.